src/App.tsx

```diff
@@ -1,7 +1,37 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends Component {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.eventHandler);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.eventHandler);
+  }
+
+  eventHandler = (event: KeyboardEvent) => {
+    const { key, code } = event;
+
+    const isSpaceKey = code === 'Space'
+      ? 'The last pressed key is [Space]'
+      : `The last pressed key is [${key}]`;
```

Comment:
No need to set the whole sentence as a pressed key value. You should store only the key as a pressedKey value